website/pages/how-to-guides/attack-surface-management-with-graph.md

```diff
@@ -0,0 +1,237 @@
+---
+title: How to use CloudQuery for Attack Surface Management and Graph Visualization
+tag: tutorial
+description: >-
+  How to use CloudQuery for Attack Surface Management (ASM) and Graph Visualization with Neo4j
+author: jsonkao
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+
+<HowToGuideHeader/>
+
+In this guide, we will demonstrate how to set up CloudQuery for customizable Attack Surface Management (ASM) and how to get started with utilizing graph visualization to execute security queries.  With cloud infrastructure and security, graph visualization can be used effectively to understand relationships between assets, attack paths, and attack surface to aid with Attack Surface Management and improving security posture of your organization.
+
+## Walkthrough
+
+### Prerequisites
+
+In this guide, we will use Neo4j as a destination and AWS as a source.  For more information on how to set those up, see our documentation on [Neo4j](https://www.cloudquery.io/docs/plugins/destinations/neo4j/overview) and [AWS](https://www.cloudquery.io/docs/plugins/sources/aws/overview).
```

Comment:
```suggestion
In this guide, we will use Neo4j as a destination and AWS as a source.  For more information on how to set those up, see our documentation on [Neo4j](/docs/plugins/destinations/neo4j/overview) and [AWS](/docs/plugins/sources/aws/overview).
```

We should not include the domain so the links point to the deploy previous domain